src/frontend/src/utils/pipes.ts

```diff
@@ -162,3 +162,13 @@ export const displayEnum = (enumString: string) => {
   enumString = enumString.charAt(0).toUpperCase() + enumString.slice(1);
   return enumString;
 };
+
+// Takes in a meeting time (0-48) and converts it to 12-clock format
+export const meetingTimePipe = (meetingTime: number) => {
```

Comment:
sorry, I think the 0-48 thing is outdated, I should've caught that in the schema comment. The way the availabilities are gonna work for this feature are 0-83 (corresponding to the 83 boxes in the components aaryan has been making, starting in the top left and going down and then left in that order).
For reference:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/3ec2e0f2-7157-453a-bdd6-beef6ad229ec)